### PR TITLE
Mpo parser

### PR DESCRIPTION
**Pull request by pnrobinson**
I am making the following change because the millisecond is not present in the dates in mp.obo and was causing an error.

-final SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
+final SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'");

### CODE REVIEW

File: ontolib-cli/pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
   <parent>
     <groupId>com.github.phenomics</groupId>
     <artifactId>OntoLib</artifactId>
-    <version>0.5-SNAPSHOT</version>
+    <version>0.3</version>
```
Comment (holtgrewe) [inline ontolib-cli/pom.xml:14]: ?!

File: ontolib-core/pom.xml [hunk 2]
```diff
@@ -11,7 +11,7 @@
   <parent>
     <groupId>com.github.phenomics</groupId>
     <artifactId>OntoLib</artifactId>
-    <version>0.5-SNAPSHOT</version>
+    <version>0.3</version>
```
Comment (holtgrewe) [inline ontolib-core/pom.xml:14]: ?!

File: ontolib-io/pom.xml [hunk 3]
```diff
@@ -11,7 +11,7 @@
   <parent>
     <groupId>com.github.phenomics</groupId>
     <artifactId>OntoLib</artifactId>
-    <version>0.5-SNAPSHOT</version>
+    <version>0.3</version>
```
Comment (holtgrewe) [inline ontolib-io/pom.xml:14]: ?!

**Review by holtgrewe — CHANGES_REQUESTED** [review]
LGTM but I think the only necessary change should be one line?

**Comment by pnrobinson** [comment]
I had been using a new version locally (0.5-SNAPSHOT) so that I could access the revised code in other maven projects. Somehow when I changed this back to 0.3 (which is the version in develop), this change got recorded here, but 0.3 is version that should get merged in; any idea how I can fix this? But I think that merging to develop should be ok?

**Pull request closed without merging**